src/send/Send.tsx

```diff
@@ -221,6 +222,10 @@ function Send({ route }: Props) {
     return null
   }
 
+  const sortedTokens = (isOutgoingPaymentRequest ? stableTokens : tokensWithBalance).sort(
+    sortFirstStableThenCeloThenOthersByUsdBalance
```

Comment:
Is this change needed for `Adding support for new tokens and search box`?  It's not apparent it's related.

`sortFirstStableThenCeloThenOthersByUsdBalance` connotes a function behavior that doesn't sound super scalable either (and I'm wondering if we should avoid using that function).

Reply:
I extract this logic from the TokenBottom sheet component.

Reply:
I think we will need to revisit how to sort when we add support for multichain